gcloud/bigtable/happybase/test_batch.py

```diff
@@ -352,7 +486,7 @@ def test_it(self):
         expected_result = [
             ['cf1', None],
             ['cf2', None],
-            ['cf3', ''],
+            ['cf3', ''],  # AGGGHHHHHHHHHH THIS IS WEIRD
```

Comment:
Yes I think you're right. Sad face. I'll send a separate PR to restore that behavior. Does the rest of this PR LGTY? (I'm going to `git push -f` with this editorial `AGGGHHHHHHHHHH THIS IS WEIRD` remove.